src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/FileTabCharacterCheckTest.java

```diff
@@ -64,21 +61,4 @@ public void testVerbose() throws Exception {
             expected);
     }
 
-    @Test
-    public void testBadFile() throws Exception {
```

Comment:
BDD was complaining about this. It was removed because it was not checking anything against the check. The error message is coming from Checker and does not even get to the check. The check does rely on external files.